subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -52,6 +54,22 @@ android {
         )
     }
 
+    buildTypes {
+        register("staging") {
+
+            initWith(named("debug").get())
+
+            setMatchingFallbacks("debug")
+
+            isMinifyEnabled = true
+            isShrinkResources = false
```

Comment:
`java.lang.ClassNotFoundException: Didn't find class "android.view.x" on path: DexPathList`
on shrink enabled, haven't dig in it yet